### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by yuanbin-dev** [chevereto/chevereto#7] [Q&A] [answered]
![chevereto_1](https://user-images.githubusercontent.com/39656612/218547006-7567d130-2509-4a4c-badb-9cb469b9cd55.jpg)
http://192.168.2.100:666/install

**Comment by rodber** [comment]
This is not a bug in Chevereto, it is a config issue on your PHP setup.

**Comment by yuanbin-dev** [comment, marked as answer]
I installed successfully by deleting the user.ini file.
